attributes/default.rb

```diff
@@ -138,14 +137,8 @@
 #                                                  #{node['hops']['home']}/share/hadoop/yarn/test/*,
 #                                                  #{node['hops']['home']}/share/hadoop/mapreduce/test/*"
 
-default['hops']['rm']['addr']                  = []
-default['hops']['rm']['http_port']             = 8088
-default['hops']['nm']['http_port']             = 8042
-default['hops']['jhs']['http_port']            = 19888
 
-#default['hops']['rm']['scheduler_class']       = "org.apache.hadoop.yarn.server.resourcemanager.scheduler.fifo.FifoScheduler"
 default['hops']['rm']['scheduler_class']       = "org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacityScheduler"
-default['hops']['rm']['scheduler_capacity']['calculator_class']  = "org.apache.hadoop.yarn.util.resource.DominantResourceCalculator"
```

Comment:
The configuration in `capacity-scheduler.xml` is going to be empty if you are not using GPUs. I guess in that case is going to take the default one. Any reason for changing from Dominant to Default?